bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -0,0 +1,9 @@
+"Mmmmm... these marshmallows are soooo good" 
+
+You close your eyes.
+
+Wait a minute....how did you end up in this car?  You must be hallucinating 
+
+[You use the gps to get back home](https://www.youtube.com/watch?v=RvlhrgSZrfA)
+
+[You open the door and there is a man sitting on your couch](../movie-ripoffs/zoolander/zoolander.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.